conpot/protocols/IEC104/IEC104_server.py

```diff
@@ -80,38 +80,29 @@ def handle(self, sock, address):
                             request += new_byte
 
                         # check if IEC 104 packet or for the first occurrence of the indication 0x68 for IEC 104
```

Comment:
This comment should be updated since you are not looking for the first occurrence anymore.